### ISSUE TITLE
SA1500 inconsistent with original StyleCop: Declaring array field with misplaced curly brackets is not reported

### ISSUE THREAD

**Issue by NikolayIT** [DotNetAnalyzers/StyleCopAnalyzers#2224]
SA1500 rule is inconsistent with original StyleCop. When I Declare an array field with misplaced curly brackets no issues are reported.

Example code:

```cs
public class Class1
{
    private int[] array =
        { 1, 2, 3 };

    public Class1()
    {
    }
}
```

The original StyleCop reports SA1500 but StyleCopAnalyzers doesn't report anything.

**Comment by NikolayIT** [comment]
Another example:

```cs
public class SomeClass
{
    private static readonly PropertyInfo[] IdentityProps =
        { typeof(SomeClass).GetTypeInfo().GetDeclaredProperty(nameof(Title)) };

    public string Title { get; set; }
}
```

Reported by StyleCop but not reported by StyleCopAnalyzers.
